spring-cloud-deployer-cloudfoundry/src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryTaskLauncher.java

```diff
@@ -127,18 +139,33 @@ public void destroy(String appName) {
 	}
 
 	@Override
-	public String getLog(String taskAppName) {
-		List<LogMessage> logMessageList = getLogMessage(taskAppName).collectList().block(Duration.ofSeconds(this.deploymentProperties.getApiTimeout()));
+	public String getLog(String id) {
```

Comment:
I can.  What is our feeling about util classes?   In years past it was frowned upon in SCDF. I don't mind doing it, but that is why I didn't implement it that way.   But if we have changed our mind on this, I'll be more than glad to do it.

Reply:
I see this not so much as a util class but rather another bean that we auto-configure than in turn gets injected the log cache cache client it is currently looking up manually. This would make it easier for future components to easily have a way to get logs when they need them. 

As far as utils classes, they are always fine by me when it makes sense. I don't typically create a util class to hold a 4 line method that is only used in 2 places, but I would for something more widely shared and a bit larger.